src/OpenTelemetry/Logs/OpenTelemetryLogger.cs

```diff
@@ -46,40 +42,17 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
                 return;
             }
 
-            var timestamp = DateTime.UtcNow;
-
-            if (state is IReadOnlyCollection<KeyValuePair<string, object>> dict)
+            var record = new LogRecord
             {
-                var isUnstructuredLog = dict.Count == 1;
+                Timestamp = DateTime.UtcNow,
+                CategoryName = this.categoryName,
+                LogLevel = logLevel,
+                EventId = eventId,
+                State = state,
+                Exception = exception,
```

Comment:
Where do we put the Scope items?
do we intent to put it ino State? (something like... this.ScopeProvider.ForEachScope(get__scope_item_and_put_into_State))

Reply:
I haven't got a chance to look into ScopeProvider, but most likely we will end up having a separate field here.